internal/cmd/vet.go

```diff
@@ -404,10 +404,6 @@ func (c *checker) fetchDatabaseUri(ctx context.Context, s config.SQL) (string, f
 		return uri, cleanup, err
 	}
 
-	if s.Engine != config.EnginePostgreSQL {
-		return "", cleanup, fmt.Errorf("managed: only PostgreSQL currently")
-	}
```

Comment:
We might still want to bail early if the engine is sqlite?

Reply:
I wanted to let the client make the request and fail so that it's easier to add engine support at the server without a client change.